material-darker.xml

```diff
@@ -0,0 +1,34 @@
+		<key name=".Vanilla" modified="2017-02-15 13:30:28" build="160724">
+			<value name="ColorTable00" type="dword" data="00151515"/>
```

Comment:
It looks like the theme is missing several common elements. 

Particularly Name, ExtendedColors, ExtendColorIdx, TextColorIdx, BackColorIdx, PopTextColorIdx, PopBackColorIdx. 

You might compare this to another one of the themes to verify - [moe-dark](https://github.com/joonro/ConEmu-Color-Themes/blob/master/moe-dark.xml)